pkg/nodeconfig/nodeconfig.go

```diff
@@ -21,6 +21,8 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	crclientcfg "sigs.k8s.io/controller-runtime/pkg/client/config"
 
+	configv1 "github.com/openshift/api/config/v1"
```

Comment:
We typically just use `config` unless there is a `v1` and `v2` in play.

Reply:
hmm, I think it doesn't work. it says `could not import github.com/openshift/api/config (no required module provides package "github.com/openshift/api/config")compilerBrokenImport`

Reply:
Sorry I meant using `config` instead of `configv1` as the import alias.